gradle/libs.versions.toml

```
@@ -22,6 +22,7 @@ androidxTestMonitor = "1.6.1"
 androidxTestUiAutomator ="2.2.0"
 androidxWork = "2.8.1"
 coil = "2.2.2"
+detekt="1.23.3"
```

```suggestion
detekt = "1.23.3"
```
To keep the same formatting across the whole file 👍

---

Done